### ISSUE TITLE
Upgrade Webpack

### ISSUE THREAD

**Issue by jaclynjessup** [Workday/canvas-kit#2618]
## 🚀 Feature Proposal

Upgrade webpack

## Motivation

To help with dependency upgrades and allowing the possibility to upgrade to Storybook7+

**Comment by jaclynjessup** [comment]
remaining work to be completed
**IN SCOPE**
- address failing tests now that we've move over to the component test runner
- globally ignore landmark tests
- aria-label on icon button

- address any issues on the Canvas Site

**OUT OF SCOPE (to be addressed at a later date)**
- address our GH actions that build storybook before cypress but we dont need this anymore with using component test runner;
so lets build storybook and run cypress in parallel

- a few styles on the storybook
